revealer_bot/tmk.py

```diff
@@ -45,9 +45,7 @@ class TMK:
     encrypted_sym_key: bytes
     conditions: Lingo
 
-    def __init__(
-        self, bulk_ciphertext: BulkCipherText, encrypted_sym_key: bytes, conditions: Lingo
-    ) -> None:
+    def __init__(self, bulk_ciphertext: BulkCipherText, encrypted_sym_key: bytes, conditions: Lingo) -> None:
```

Comment:
Just formatting changes stemming from increasing max-line-length to 160 characters